components/BackButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { useRouter } from "next/router";
+import { Button } from "@mantine/core";
+
+/**
+ * BackButton is a component for rendering a back button
+ * @returns Button that routes user back when clicked
+ */
+const BackButton = () => {
+  const router = useRouter();
+  return (
+    <Button
+      onClick={() => router.back()}
+      color="cyan"
+      radius="md"
+      size="sm"
+      variant="filled"
+      style={{
+        float: "left",
+      }}
+    >
+      Back
```

Comment:
Could be *"fun"* to render a back arrow icon here instead of text. Wanna give it a shot and see what you think looks better?